transport.rst

```diff
@@ -1,16 +1,26 @@
 Transport
 #########
 
+Summary
+============
+
+Raiden is a network agnostic protocol. Protocol messages can in general be transferred over
```

Comment:
It is true that the network itself is transport-agnostic, but we should make clear here that Matrix is the only transport option of the reference implementation, so to be able to join the existing/evolving Raiden Network a client needs to speak Matrix.

Reply:
Added.